_includes/field-description-description.md

```diff
@@ -1,12 +1,12 @@
-{%- assign payment_instrument = include.payment_instrument -%}
-{%- if payment_instrument == "mobilepay"-%}
-    {%- assign payment_instrument ="mobile-pay" -%}
+{%- assign documentation_section = include.documentation_section -%}
+{%- if documentation_section == "mobilepay"-%}
+    {%- assign documentation_section ="mobile-pay" -%}
```

Comment:
The purpose of this task is to remove such checks, if needed you should pass to parameters to the include, one for the `documentation_section` and one for the `api_resource`